pkg/docker/docker.go

```diff
@@ -347,16 +347,26 @@ func (d *stiDocker) GetOnBuild(name string) ([]string, error) {
 // and returns the image metadata
 func (d *stiDocker) CheckAndPullImage(name string) (*docker.Image, error) {
 	name = getImageName(name)
+	displayName := name
+
+	if glog.Is(3) == false {
```

Comment:
@php-coder It's comments on this issue: https://github.com/openshift/origin/issues/9247

glog.Is(x) really means loglevel >= x ... not very well named, but it's used correctly above.

@jupierce why not  if !glog.Is(3) ?